.eslintignore

```diff
@@ -5,4 +5,5 @@ gatsby-types.d.ts
 package.json
 .eslintrc.json
 .vscode
-tsconfig.json
\ No newline at end of file
+tsconfig.json
+README.me
```

Comment:
Why README.me?

Reply:
i was getting eslint errors on the README and some other types of files, but when i committed it, it seemed to be fine?? so honestly that isn't necessary